aws-typescript/package.json

```diff
@@ -4,8 +4,8 @@
         "@types/node": "latest"
     },
     "dependencies": {
-        "@pulumi/pulumi": "^0.17.28",
-        "@pulumi/aws": "^0.18.27",
+        "@pulumi/pulumi": "^1.0.0",
+        "@pulumi/aws": "^1.0.0",
         "@pulumi/awsx": "^0.18.8"
```

Comment:
Nit - should this be ^0.18.10 like in the js version of this template?

Reply:
Fixed - nice catch!